operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -72,6 +71,27 @@ spec:
 # Choose "anonymous" to allow full access to Kiali without requiring any credentials.
 # Choose "openshift" to use the OpenShift OAuth login which controls access based on
 # the individual's  RBAC roles in OpenShift. Not valid for non-OpenShift environments.
+# Choose "ldap" to enable LDAP based authentication. There are additional configurations for
+# under ldap in the yaml file. The configurations for LDAP are as follows
+#    ---
+#    strategy: ldap
+#    ldap:
+#       # AD IP host and port
```

Comment:
This "AD" will confuse LDAP users that do not use ActiveDirectory (I assume this is what you mean by "AD"). Plus, I think its pretty obvious what "ldap_host" and "ldap_port" are - so no need to comment this. Remove this line.
```suggestion
```

Reply:
Removed the comment